fdbserver/TransactionTagCounter.cpp

```diff
@@ -90,12 +90,17 @@ class TransactionTagCounterImpl {
 	std::vector<StorageQueuingMetricsReply::TagInfo> previousBusiestTags;
 	Reference<EventCacheHolder> busiestReadTagEventHolder;
 
+	const std::string busiestWriteTagTrackingKey;
+	Reference<EventCacheHolder> busiestWriteTagEventHolder;
```

Comment:
Does `TransactionTagCounter` have anything to do with the busiest write tag accounting? If not, it might be clearer to move this event holder to the storage server instead.